### DISCUSSION TITLE
Recently played endpoint

### DISCUSSION THREAD

**Discussion by RebellionBritney** [tidal-music/discussions#96] [Ideas & Feature requests]
I want to work in an app where you can share with other users their listening activity
In the API I can't find a way to get the recently played endpoint
Any suggestion? Is that even available yet? Or maybe get access in the history of the user played music?

**Comment by stefanmoro** [comment]
Hey @RebellionBritney,
We currently don't expose current/recently played API. It has been discussed and requested in [this conversation](https://github.com/orgs/tidal-music/discussions/10), so I'll close this as a duplicate.
